embassy/Cargo.toml

```diff
@@ -40,6 +40,9 @@ defmt-timestamp-uptime = ["defmt"]
 # Enabling it directly without supplying a time driver will fail to link.
 time = []
 
+# enable the Xtensa executor
+xtensa = []
```

Comment:
this feature is unused (there's no `cfg(feature="xtensa")` anywhere in the code).

Instead, could you make it so that when building for xtensa, you get the executor if `nightly` is enabled, and you don't get it otherwise? There's no need to have a separate feature.